easybuild/easyconfigs/l/libnsl/libnsl-1.3.0-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'libnsl'
+version = '1.3.0'
+
+homepage = 'https://www.linuxfromscratch.org/blfs/view/svn/basicnet/libnsl.html'
+description = """The libnsl package contains the public client interface for NIS(YP)."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/thkukuk/%(name)s/releases/download/v%(version)s/']
+sources = [SOURCE_TAR_XZ]
+
```

Comment:
missing checksum

Reply:
Oops, fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/14494/commits/810ef3e0f2ee3eacdcdbf04d73606757dd0560fc.